go-controller/pkg/ovn/external_ids_syncer/port_group/port_group_sync.go

```diff
@@ -98,6 +110,12 @@ func (syncer *PortGroupSyncer) getReferencingObjsAndNewDbIDs(oldHash, oldName, n
 	// Filter port groups with pre-defined names
 	case oldName == types.ClusterPortGroupNameBase || oldName == types.ClusterRtrPortGroupNameBase:
 		dbIDs = getPortGroupClusterDbIDs(oldName, networkExternalID)
+	case strings.HasPrefix(oldName, "ANP:"):
```

Comment:
is this to take care of upgrades and cleanup/update old set of PGs?
I would vote not to do this and avoid having to mantain this code specially because we just added support for this feature. I doubt anybody is using this upstream :) +1 to quickly get this in haha and pretend like this is how it was :D not bother about syncing up old names here since this feature is not that old.

Reply:
I see your point, but considering we already have this syncer, adding ANP is just a little bit of code, so I though if we can easily make it upgradable, why not.
More that that, we need to have this extra case for (B)ANP in switch because our default case is namespace port group.